modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration is done, there are necessary steps to clean up resources created by Kuryr. From high level perspective it will include:
+
+* Removing Kuryr finalizers from resources
+* Removing Kuryr CRDs
+* Removing {rh-openstack-first} resources
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install `jq` tool
```

Comment:
We don't typically allow `jq` in the official docs. Can we work around this?

Reply:
Well, that's a bummer, as we heavily relay on that tool.

Reply:
ok, I figure that out, and remove it from the requirements.